### DISCUSSION TITLE
Zebar

### DISCUSSION THREAD

**Discussion by Entharia** [catppuccin/catppuccin#2500] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the suggested port be themed?

[Can be styled with css](https://github.com/glzr-io/zebar?tab=readme-ov-file#concept-1-styled-with-html--css)

### Any additional comments?

Zebar is now used in glazewm which makes the original glazewm port kinda obsolete
https://github.com/glzr-io/glazewm

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by gamersekofy** [comment]
Thank you for this! Eagerly waiting for your port :)

**Comment by fathulfahmy** [comment]
I have mentioned this on https://github.com/catppuccin/glazewm/issues/3. I will however be working on [zebar](https://github.com/glzr-io/zebar) starting this week and expected to deliver latest by next week (22/3/2025). General idea is to map colors to CSS variables.

**Reply by fathulfahmy** [reply]
Progress update: Pending review as CSS port #2737

**Reply by fathulfahmy** [reply]
Progress update: Canceled port since it's a duplicate. Use https://github.com/catppuccin/palette/blob/main/docs/css.md

**Reply by fathulfahmy** [reply]
Progress update: 
1. Updated [catppuccin/glazewm](https://github.com/catppuccin/glazewm) for [GlazeWM 3.0](https://github.com/glzr-io/glazewm) and [Zebar](https://github.com/glzr-io/zebar#-getting-started). 
2. Created a port for YAML ([catppuccin/yaml](https://github.com/catppuccin/catppuccin/issues/2857))
